pacsman/filesystem_dev_client.py

```diff
@@ -127,6 +127,21 @@ def fetch_thumbnail(self, series_id):
         for (path, ds) in dicom_datasets.items():
             if ds.SeriesInstanceUID == series_id:
                 series_paths.append(path)
-        # TODO check sorting of IDs
-        if series_paths:
-            shutil.copy(series_paths[len(series_paths) // 2], self.dicom_dir)
+        if not series_paths:
+            return None
+
+        thumbnail_series_path = series_paths[len(series_paths) // 2]
+        shutil.copy(thumbnail_series_path, self.dicom_dir)
```

Comment:
Why do we need to copy this file here?

Reply:
For the filesystem client, this is what constitutes the "fetch" or C-MOVE. All the stuff in `series_path` is the "data store" and the `self.dicom_dir` is the same output directory that all `DicomInterface` classes would use.

Reply:
Ahh, that makes sense.  Thanks for the clarification!